app/authConfig.js

```diff
@@ -0,0 +1,30 @@
+// Config object to be passed to Msal on creation.
+// For a full list of msal.js configuration parameters, 
+// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_authenticationparameters_.html
+const msalConfig = {
+    auth: {
+        clientId: "Enter_the_Application_Id_Here",
+        authority: "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here",
+        redirectUri: "Enter_the_Redirect_Uri_Here",
+    },
+    cache: {
+        cacheLocation: "sessionStorage", // This configures where your cache will be stored
+        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    }
+};
+
+// Add here scopes for id token to be used at MS Identity Platform endpoints.
+const loginRequest = {
+    scopes: ["openid", "profile", "User.Read"]
+};
+
+// Add here scopes for access token to be used at MS Graph API endpoints.
+const tokenRequest = {
+    scopes: ["Mail.Read"]
+};
+
+// Add here scopes for token to be fetched silently
+const silentRequest = {
+    scopes: ["Enter_the_Application_Id_Here", "User.Read", "Mail.Read"],
```

Comment:
Can we just do this like: 
```
const silentRequest = {
           scopes: [msalConfig.auth.clientId, "User.Read", "Mail.Read"],
```
in order to avoid entering clientId twice (that might increase the chance of mistakes)

Reply:
done